tests/test_verify.py

```diff
@@ -0,0 +1,6 @@
+from src.verify import verify
+
+def test_verify() -> None:
+    list_numbers = ('1','2','3','4','5','6','7','8','9','10')
```

Comment:
you could declare this `list_numbers` outside of the `main()` function, import it here and re-use it